### ISSUE TITLE
why your model input is 225

### ISSUE THREAD

**Issue by lmmcc** [shicai/SENet-Caffe#5]
hello, i want to know why your input is 225,while the original one is 224,

**Comment by shicai** [comment]
simply because i don't like the floor/ceil mode of conv7x7/s2 and pool3x3/s2 in caffe.
```
for input size = 225:
output size of conv7x7/s2 = (225 - 7 + 2*3) / 2 + 1 = 113,
output size of pool3x3/s2 = (113 - 3) / 2 + 1 = 56.

while for input size = 224:
that is floor((224 - 7 + 2*3) / 2) + 1 = 112, and 
ceil((112 - 3) / 2) + 1 = 56.
```

**Comment by lmmcc** [comment]
thanks for your reply,,,sorry for another question,you provide a test prototxt,in your proto,,the bn layer's batch_norm_param is set to eps:1e-4，and have another three params，will it be changed to "use_global_stats:true" in test phase?

**Comment by shicai** [comment]
yes. it is automatically configured by caffe: in train phase, `use_global_stats` is false, and in test phase, it is true.
